skimage/util/apply_parallel.py

```diff
@@ -90,8 +90,9 @@ def apply_parallel(function, array, chunks=None, depth=0, mode=None,
     extra_keywords : dictionary, optional
         Dictionary of keyword arguments to be passed to the function.
     compute : bool, optional
-        Whether to compute right away (default) or
-        skip computing and return a dask Array.
+        Whether to compute eagerly or lazily (returning a Dask Array).
```

Comment:
From this docstring it is not clear what `True` and `False` for `compute` actually do. Could you elaborate on the correspondence, please?

Reply:
what he said. ⬆️ 😛

Reply:
Feel free to suggest things here. ;) Had a hard time capturing this concisely.

Reply:
So I tried something. If it works or needs minor fixes, happy to clean up as needed. If it's way off, please suggest what you would like to see here instead. :)